LibreNMS/OS/Ray.php

```diff
@@ -72,11 +72,11 @@ public function discoverProcessors()
     public function discoverWirelessFrequency()
     {
 
-        $oid_tx = '.1.3.6.1.4.1.33555.1.2.1.4'; // RAY-MIB::txFreq.0
-        $oid_rx = '.1.3.6.1.4.1.33555.1.2.1.4'; // RAY-MIB::rxFreq.0
-        $cmd = gen_snmpget_cmd($this->getDevice(), $oid_tx, '-Oqv', 'RAY-MIB', 'ray');
-        $data = trim(external_exec($cmd), "\" \n\r");
-        if (!preg_match('/(No Such Instance)/i', $data)) {
+        $oid_tx = '.1.3.6.1.4.1.33555.1.2.1.4'; // RAY-MIB::txFreq
+        $oid_rx = '.1.3.6.1.4.1.33555.1.2.1.4'; // RAY-MIB::rxFreq
+
+        $cmd = snmp_get($this->getDevice(), $oid_tx, '-Oqv', 'RAY-MIB', 'ray');
+        if ($cmd !== false) {
             return array(
```

Comment:
Whey even do the snmpget at all?  The Wireless code will actually do an snmpget and verify it has data before adding the device.

Reply:
The problem is this:

Old firmware:
An OID without .0 at the end exits and returns a value.
An OID with .0 at the end exists and returns a value.

New firmware:
An OID without .0 does not exist, it returns an error.
An OID with .0 exists and returns a value.

If you suggest another solution, I will be happy to implement it.

Reply:
So, you should always set it to .0 right?